dts/arc/tenstorrent/tt_blackhole_fixed_partitions.dtsi

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2025 Tenstorrent AI ULC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+/ {
+	partitions {
+		compatible = "tenstorrent,fixed-partitions";
```

Comment:
Are there any gains by using `tenstorrent` instead of `zephyr` for the vendor in this compatible? I think we should be able to use the upstream one.

Reply:
Daniel said we want custom properties like `executable` and `padto` which aren't defined upstream